tsdb/index/postings.go

```diff
@@ -79,6 +79,56 @@ func (p *MemPostings) SortedKeys() []labels.Label {
 	return keys
 }
 
+type PostingsStats struct {
```

Comment:
Exported type should have a godoc comment. Perhaps `// PostingsStats contains cardinality based statistics for postings.`